cannot personalize path for config file in nu shell

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

### What happened?

after reading guide on https://ohmyposh.dev/docs/installation/customize#set-the-configuration

this command doesn't work as expected:
```
oh-my-posh init nu --config ~/powerlevel10k_rainbow.omp.json --print | save ~/.oh-my-posh.nu --force
```

the file .oh-my-posh.nu is created empty and the real config is created in "vendor/autoload" default path of nushell with name init.26.8.0.2a2f4f2e0f4fa03d.10.nu

### Theme

[powerlevel10k_rainbow](https://github.com/JanDeDobbeleer/oh-my-posh/blob/main/themes/powerlevel10k_rainbow.omp.json)

### What OS are you seeing the problem on?

macOS

### Which shell are you using?

nu

### Log output

```Shell
Version: 26.8.0

Shell: nu (0.105.1)

Prompt:

╭─    ~                                                                                                                                                                                                                  20:54:56  ─╮
╰─                                                                                                                                                                                                                                           ─╯

Segments:

ConsoleTitle(true)                         -   0 ms
Os(true)                                   -   0 ms
Path(true)                                 -   1 ms
Git(false)                                 -   1 ms
Node(false)                                -   0 ms
Go(false)                                  -   0 ms
Julia(false)                               -   0 ms
Python(false)                              -   0 ms
Ruby(false)                                -   0 ms
Azfunc(false)                              -   0 ms
Aws(false)                                 -   0 ms
Root(false)                                -   0 ms
Executiontime(false)                       -   0 ms
Status(true)                               -   0 ms
Time(true)                                 -   0 ms
Text(true)                                 -   0 ms
Text(true)                                 -   0 ms

Run duration: 19.00375ms

Cache path: /Users/valentino/.cache/oh-my-posh

Config path: /opt/homebrew/opt/oh-my-posh/themes/powerlevel10k_rainbow.omp.json

Logs:

[DEBUG] 20:54:56.809 debug.go:36 → debug mode enabled
[DEBUG] 20:54:56.809 load.go:Path:79 → config set using POSH_THEME: /opt/homebrew/opt/oh-my-posh/themes/powerlevel10k_rainbow.omp.json
[TRACE] 20:54:56.809 load.go:Path() - 341.791µs
[TRACE] 20:54:56.810 load.go:loadConfig() - 1.128792ms
[TRACE] 20:54:56.810 load.go:Load() - 1.133959ms
[DEBUG] 20:54:56.810 terminal.go:Init:54 → plain mode enabled
[DEBUG] 20:54:56.811 home.go:12 → /Users/valentino
[TRACE] 20:54:56.811 path.go:Path() - 288.667µs
[DEBUG] 20:54:56.811 file.go:Init:26 → loading cache file: /Users/valentino/.cache/oh-my-posh/omp.cache
[DEBUG] 20:54:56.811 file.go:Init:47 → loading cache key: INITVERSIONZSH
[DEBUG] 20:54:56.811 file.go:Init:47 → loading cache key: NUAUTOLOADDIR
[DEBUG] 20:54:56.811 file.go:Init:47 → loading cache key: is_wsl
[DEBUG] 20:54:56.811 file.go:Init:47 → loading cache key: INITVERSIONNU
[TRACE] 20:54:56.811 file.go:Init(/Users/valentino/.cache/oh-my-posh/omp.cache) - 258.958µs
[DEBUG] 20:54:56.811 home.go:12 → /Users/valentino
[TRACE] 20:54:56.811 path.go:Path() - 11µs
[DEBUG] 20:54:56.811 file.go:Init:26 → loading cache file: /Users/valentino/.cache/oh-my-posh/omp.cache.5cda5613-aa22-4c0b-864d-78771d33ae10
[DEBUG] 20:54:56.811 file.go:Init:47 → loading cache key: prompt_count_cache
[TRACE] 20:54:56.811 file.go:Init(/Users/valentino/.cache/oh-my-posh/omp.cache.5cda5613-aa22-4c0b-864d-78771d33ae10) - 150.459µs
[DEBUG] 20:54:56.811 file.go:Get:80 → getting cache key: prompt_count_cache with value: 2
[TRACE] 20:54:56.811 terminal.go:setPromptCount() - 13.875µs
[TRACE] 20:54:56.811 terminal.go:GOOS() - 42ns
[DEBUG] 20:54:56.811 terminal.go:setPwd:118 → /Users/valentino
[TRACE] 20:54:56.811 terminal.go:setPwd() - 22.916µs
[TRACE] 20:54:56.811 terminal.go:Init() - 1.142416ms
[TRACE] 20:54:56.811 terminal.go:Shell() - 83ns
[TRACE] 20:54:56.811 terminal.go:Flags() - 42ns
[DEBUG] 20:54:56.811 file.go:Get:75 → cache key not found: template_cache
[TRACE] 20:54:56.811 cache.go:restoreCache() - 2.708µs
[TRACE] 20:54:56.811 terminal_unix.go:Root() - 1.083µs
[TRACE] 20:54:56.811 terminal.go:Shell() - 0s
[TRACE] 20:54:56.811 terminal.go:Flags() - 42ns
[TRACE] 20:54:56.811 terminal.go:StatusCodes() - 417ns
[DEBUG] 20:54:56.811 file.go:Get:80 → getting cache key: is_wsl with value: false
[DEBUG] 20:54:56.811 terminal_unix.go:IsWsl:32 → false
[TRACE] 20:54:56.811 terminal_unix.go:IsWsl() - 5.25µs
[TRACE] 20:54:56.812 terminal.go:Flags() - 41ns
[TRACE] 20:54:56.812 terminal.go:Flags() - 0s
[DEBUG] 20:54:56.812 home.go:12 → /Users/valentino
[DEBUG] 20:54:56.812 file.go:Get:80 → getting cache key: is_wsl with value: false
[DEBUG] 20:54:56.812 terminal_unix.go:IsWsl:32 → false
[TRACE] 20:54:56.812 terminal_unix.go:IsWsl() - 3.541µs
[TRACE] 20:54:56.812 terminal.go:Flags() - 42ns
[TRACE] 20:54:56.812 terminal.go:Flags() - 0s
[TRACE] 20:54:56.812 terminal.go:GOOS() - 0s
[DEBUG] 20:54:56.812 terminal.go:User:252 → valentino
[TRACE] 20:54:56.812 terminal.go:User() - 1.875µs
[DEBUG] 20:54:56.812 terminal.go:Host:269 → Mac.fritz.box
[TRACE] 20:54:56.812 terminal.go:Host() - 7.833µs
[TRACE] 20:54:56.812 terminal.go:GOOS() - 42ns
[DEBUG] 20:54:56.812 terminal.go:Getenv:80 → 1
[TRACE] 20:54:56.812 terminal.go:Getenv(SHLVL) - 1.959µs
[DEBUG] 20:54:56.812 writer.go:Init:106 → terminal program: ghostty
[DEBUG] 20:54:56.812 writer.go:Init:107 → terminal shell: shell
[DEBUG] 20:54:56.812 terminal.go:Getenv:80 → NO DATA
[TRACE] 20:54:56.812 terminal.go:Getenv(OMP_CACHE_DISABLED) - 2.334µs
[DEBUG] 20:54:56.812 file.go:Get:75 → cache key not found: accent_color
[ERROR] 20:54:56.820 terminal.go:RunCommand:293 → exit status 1
[DEBUG] 20:54:56.820 terminal.go:RunCommand:296 ↓
    2025-06-14 20:54:56.819 defaults[85924:8913717]
    The domain/default pair of (kCFPreferencesAnyApplication, AppleAccentColor) does not exist

[TRACE] 20:54:56.820 terminal.go:RunCommand(defaults read -g AppleAccentColor) - 7.803917ms
[ERROR] 20:54:56.820 colors_darwin.go:GetAccentColor:14 → exit status 1
[TRACE] 20:54:56.820 colors.go:SetAccentColor() - 7.839208ms
[TRACE] 20:54:56.820 terminal.go:Shell() - 83ns
[DEBUG] 20:54:56.820 terminal.go:Getenv:80 → 0.105.1
[TRACE] 20:54:56.820 terminal.go:Getenv(POSH_SHELL_VERSION) - 3.208µs
[DEBUG] 20:54:56.820 debug.go:PrintDebug:24 → segment: Title
[DEBUG] 20:54:56.820 text.go:patchTemplate:125 → {{ .Shell }} in {{ .Folder }}
[TRACE] 20:54:56.820 text.go:Render({{ .Shell }} in {{ .Folder }}) - 247.625µs
[TRACE] 20:54:56.820 terminal.go:Flags() - 42ns
[DEBUG] 20:54:56.820 terminal.go:Getenv:80 → NO DATA
[TRACE] 20:54:56.820 terminal.go:Getenv(POSH_CURSOR_LINE) - 2.25µs
[DEBUG] 20:54:56.820 terminal.go:Getenv:80 → NO DATA
[TRACE] 20:54:56.820 terminal.go:Getenv(POSH_CURSOR_COLUMN) - 1.333µs
[TRACE] 20:54:56.820 terminal.go:Flags() - 0s
[TRACE] 20:54:56.820 terminal.go:Flags() - 42ns
[TRACE] 20:54:56.821 terminal.go:Flags() - 42ns
[TRACE] 20:54:56.821 terminal.go:Flags() - 125ns
[TRACE] 20:54:56.821 terminal.go:Flags() - 84ns
[DEBUG] 20:54:56.821 home.go:12 → /Users/valentino
[DEBUG] 20:54:56.821 home.go:12 → /Users/valentino
[TRACE] 20:54:56.821 terminal.go:GOOS() - 83ns
[TRACE] 20:54:56.821 terminal.go:GOOS() - 83ns
[DEBUG] 20:54:56.821 home.go:12 → /Users/valentino
[DEBUG] 20:54:56.821 segment.go:Execute:112 → segment: Path
[DEBUG] 20:54:56.821 segment.go:Execute:112 → segment: Os
[TRACE] 20:54:56.821 terminal.go:GOOS() - 0s
[DEBUG] 20:54:56.821 segment.go:Execute:112 → segment: Git
[DEBUG] 20:54:56.821 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 20:54:56.821 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 20:54:56.821 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 20:54:56.821 segment.go:isToggled:233 → no toggles found
[DEBUG] 20:54:56.821 segment.go:isToggled:233 → no toggles found
[DEBUG] 20:54:56.821 segment.go:isToggled:233 → no toggles found
[TRACE] 20:54:56.821 terminal.go:GOOS() - 83ns
[DEBUG] 20:54:56.821 properties.go:GetString:28 → macos: 
[DEBUG] 20:54:56.821 text.go:patchTemplate:125 →  {{ if .WSL }}WSL at {{ end }}{{.Data.Icon}}
[DEBUG] 20:54:56.822 properties.go:GetBool:22 → display_cygpath: false
[TRACE] 20:54:56.822 terminal.go:GOOS() - 42ns
[TRACE] 20:54:56.822 separator.go:Separator() - 41ns
[TRACE] 20:54:56.822 terminal.go:Shell() - 84ns
[TRACE] 20:54:56.822 terminal.go:Shell() - 41ns
[DEBUG] 20:54:56.822 file.go:Get:80 → getting cache key: is_wsl with value: false
[DEBUG] 20:54:56.822 terminal_unix.go:IsWsl:32 → false
[TRACE] 20:54:56.822 terminal_unix.go:IsWsl() - 296.833µs
[TRACE] 20:54:56.822 terminal_unix.go:IsWsl2() - 298.959µs
[TRACE] 20:54:56.822 terminal.go:GOOS() - 42ns
[DEBUG] 20:54:56.822 terminal.go:CommandPath:320 → /usr/bin/git
[TRACE] 20:54:56.822 terminal.go:CommandPath(git) - 70.041µs
[TRACE] 20:54:56.822 terminal.go:HasCommand(git) - 72.292µs
[DEBUG] 20:54:56.822 properties.go:GetBool:22 → fetch_bare_info: false
[DEBUG] 20:54:56.822 terminal.go:ResolveSymlink:211 → /Users/valentino
[TRACE] 20:54:56.822 terminal.go:ResolveSymlink(/Users/valentino) - 10.292µs
[ERROR] 20:54:56.822 terminal.go:HasParentFilePath:479 → stat .git: no such file or directory
[TRACE] 20:54:56.822 terminal.go:HasParentFilePath(.git) - 22.042µs
[TRACE] 20:54:56.822 text.go:Render( {{ if .WSL }}WSL at {{ end }}{{.Icon}} ) - 420µs
[TRACE] 20:54:56.822 text.go:Render(diamond) - 125ns
[TRACE] 20:54:56.822 terminal.go:GOOS() - 42ns
[DEBUG] 20:54:56.822 properties.go:GetBool:22 → mapped_locations_enabled: true
[DEBUG] 20:54:56.822 properties.go:GetString:28 → windows_registry_icon: 
[DEBUG] 20:54:56.822 properties.go:GetString:28 → windows_registry_icon: 
[DEBUG] 20:54:56.822 home.go:12 → /Users/valentino
[TRACE] 20:54:56.822 separator.go:Separator() - 41ns
[TRACE] 20:54:56.822 terminal.go:GOOS() - 42ns
[TRACE] 20:54:56.822 terminal.go:GOOS() - 42ns
[DEBUG] 20:54:56.822 properties.go:GetString:28 → home_icon: ~
[DEBUG] 20:54:56.822 properties.go:GetKeyValueMap:46 → mapped_locations: map[]
[TRACE] 20:54:56.822 separator.go:Separator() - 0s
[TRACE] 20:54:56.822 terminal.go:GOOS() - 42ns
[TRACE] 20:54:56.822 terminal.go:GOOS() - 42ns
[TRACE] 20:54:56.822 separator.go:Separator() - 42ns
[TRACE] 20:54:56.822 terminal.go:GOOS() - 0s
[TRACE] 20:54:56.822 terminal.go:GOOS() - 42ns
[TRACE] 20:54:56.822 terminal.go:GOOS() - 41ns
[TRACE] 20:54:56.822 terminal.go:GOOS() - 41ns
[TRACE] 20:54:56.822 terminal.go:GOOS() - 0s
[TRACE] 20:54:56.822 terminal.go:Flags() - 0s
[DEBUG] 20:54:56.822 properties.go:GetStringArray:52 → cycle: []
[DEBUG] 20:54:56.822 properties.go:GetString:28 → folder_separator_template:
[DEBUG] 20:54:56.822 properties.go:GetString:28 → folder_separator_icon: /
[DEBUG] 20:54:56.822 properties.go:GetBool:22 → cycle_folder_separator: false
[DEBUG] 20:54:56.822 properties.go:GetString:28 → folder_format: %s
[DEBUG] 20:54:56.822 properties.go:GetString:28 → edge_format: %s
[DEBUG] 20:54:56.822 properties.go:GetString:28 → left_format: %s
[DEBUG] 20:54:56.822 properties.go:GetString:28 → right_format: %s
[TRACE] 20:54:56.822 terminal.go:Flags() - 42ns
[TRACE] 20:54:56.822 terminal.go:GOOS() - 42ns
[TRACE] 20:54:56.822 terminal.go:StackCount() - 83ns
[TRACE] 20:54:56.822 terminal_unix.go:DirIsWritable(/Users/valentino) - 12.666µs
[DEBUG] 20:54:56.822 text.go:patchTemplate:125 →   {{ .Data.Path }}
[TRACE] 20:54:56.822 text.go:Render(  {{ .Path }} ) - 40.209µs
[TRACE] 20:54:56.822 text.go:Render(powerline) - 125ns
[TRACE] 20:54:56.822 text.go:Render(powerline) - 84ns
[TRACE] 20:54:56.822 terminal.go:Shell() - 41ns
[TRACE] 20:54:56.822 terminal.go:Shell() - 42ns
[TRACE] 20:54:56.822 terminal.go:Flags() - 42ns
[TRACE] 20:54:56.822 terminal.go:Flags() - 83ns
[DEBUG] 20:54:56.822 home.go:12 → /Users/valentino
[TRACE] 20:54:56.822 terminal.go:GOOS() - 41ns
[TRACE] 20:54:56.822 terminal.go:Flags() - 41ns
[DEBUG] 20:54:56.822 segment.go:Execute:112 → segment: Time
[DEBUG] 20:54:56.822 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 20:54:56.822 home.go:12 → /Users/valentino
[DEBUG] 20:54:56.822 segment.go:isToggled:233 → no toggles found
[TRACE] 20:54:56.822 terminal.go:GOOS() - 42ns
[TRACE] 20:54:56.822 terminal.go:Flags() - 83ns
[DEBUG] 20:54:56.822 segment.go:Execute:112 → segment: Go
[DEBUG] 20:54:56.822 properties.go:GetString:28 → time_format: 15:04:05
[DEBUG] 20:54:56.822 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 20:54:56.822 home.go:12 → /Users/valentino
[DEBUG] 20:54:56.822 segment.go:isToggled:233 → no toggles found
[TRACE] 20:54:56.822 terminal.go:Flags() - 42ns
[TRACE] 20:54:56.822 terminal.go:GOOS() - 42ns
[DEBUG] 20:54:56.822 segment.go:Execute:112 → segment: Node
[DEBUG] 20:54:56.822 home.go:12 → /Users/valentino
[DEBUG] 20:54:56.822 home.go:12 → /Users/valentino
[DEBUG] 20:54:56.822 file.go:Get:75 → cache key not found: toggle_cache
[TRACE] 20:54:56.822 terminal.go:GOOS() - 0s
[TRACE] 20:54:56.822 terminal.go:GOOS() - 0s
[DEBUG] 20:54:56.822 segment.go:isToggled:233 → no toggles found
[DEBUG] 20:54:56.822 segment.go:Execute:112 → segment: Azfunc
[DEBUG] 20:54:56.822 segment.go:Execute:112 → segment: Aws
[DEBUG] 20:54:56.822 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 20:54:56.822 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 20:54:56.822 segment.go:isToggled:233 → no toggles found
[DEBUG] 20:54:56.822 segment.go:isToggled:233 → no toggles found
[DEBUG] 20:54:56.822 properties.go:GetBool:22 → display_default: false
[DEBUG] 20:54:56.822 terminal.go:Getenv:80 → NO DATA
[TRACE] 20:54:56.822 terminal.go:Getenv(AWS_VAULT) - 2.375µs
[DEBUG] 20:54:56.822 terminal.go:Getenv:80 → NO DATA
[TRACE] 20:54:56.822 terminal.go:Getenv(AWS_DEFAULT_PROFILE) - 1.667µs
[DEBUG] 20:54:56.822 terminal.go:Getenv:80 → NO DATA
[TRACE] 20:54:56.822 terminal.go:Getenv(AWS_PROFILE) - 1.292µs
[DEBUG] 20:54:56.822 terminal.go:Getenv:80 → NO DATA
[TRACE] 20:54:56.822 terminal.go:Getenv(AWS_REGION) - 1.666µs
[DEBUG] 20:54:56.822 terminal.go:Getenv:80 → NO DATA
[TRACE] 20:54:56.822 terminal.go:Getenv(AWS_DEFAULT_REGION) - 1.375µs
[DEBUG] 20:54:56.822 terminal.go:Getenv:80 → NO DATA
[DEBUG] 20:54:56.822 properties.go:GetStringArray:52 → extensions: [*.go go.mod go.sum go.work go.work.sum]
[TRACE] 20:54:56.822 terminal.go:Getenv(AWS_CONFIG_FILE) - 1.792µs
[TRACE] 20:54:56.822 terminal.go:Flags() - 42ns
[DEBUG] 20:54:56.822 properties.go:GetStringArray:52 → folders: []
[DEBUG] 20:54:56.822 home.go:12 → /Users/valentino
[DEBUG] 20:54:56.822 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 20:54:56.822 home.go:12 → /Users/valentino
[DEBUG] 20:54:56.822 properties.go:GetStringArray:52 → extensions: [host.json local.settings.json function.json]
[TRACE] 20:54:56.822 terminal.go:GOOS() - 42ns
[DEBUG] 20:54:56.822 properties.go:GetStringArray:52 → folders: []
[DEBUG] 20:54:56.822 segment.go:Execute:112 → segment: Executiontime
[DEBUG] 20:54:56.822 properties.go:GetBool:22 → home_enabled: false
[ERROR] 20:54:56.822 terminal.go:FileContent:223 → open /Users/valentino/.aws/config: no such file or directory
[DEBUG] 20:54:56.822 file.go:Get:75 → cache key not found: toggle_cache
[TRACE] 20:54:56.822 terminal.go:FileContent(/Users/valentino/.aws/config) - 13.166µs
[DEBUG] 20:54:56.822 segment.go:isToggled:233 → no toggles found
[DEBUG] 20:54:56.822 home.go:12 → /Users/valentino
[DEBUG] 20:54:56.822 home.go:12 → /Users/valentino
[TRACE] 20:54:56.822 terminal.go:Flags() - 42ns
[TRACE] 20:54:56.822 terminal.go:Flags() - 42ns
[DEBUG] 20:54:56.822 home.go:12 → /Users/valentino
[DEBUG] 20:54:56.822 home.go:12 → /Users/valentino
[TRACE] 20:54:56.822 terminal.go:GOOS() - 41ns
[TRACE] 20:54:56.822 terminal.go:GOOS() - 0s
[DEBUG] 20:54:56.822 segment.go:Execute:112 → segment: Python
[DEBUG] 20:54:56.822 segment.go:Execute:112 → segment: Ruby
[DEBUG] 20:54:56.822 segment.go:Execute:112 → segment: Root
[DEBUG] 20:54:56.822 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 20:54:56.822 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 20:54:56.822 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 20:54:56.822 segment.go:isToggled:233 → no toggles found
[DEBUG] 20:54:56.822 segment.go:isToggled:233 → no toggles found
[DEBUG] 20:54:56.822 properties.go:GetBool:22 → always_enabled: false
[DEBUG] 20:54:56.822 segment.go:isToggled:233 → no toggles found
[TRACE] 20:54:56.822 terminal.go:ExecutionTime() - 42ns
[DEBUG] 20:54:56.822 properties.go:GetFloat64:34 → threshold: 500.000000
[DEBUG] 20:54:56.822 properties.go:GetStringArray:52 → extensions: [*.js *.ts package.json .nvmrc pnpm-workspace.yaml .pnpmfile.cjs .vue]
[DEBUG] 20:54:56.822 properties.go:GetStringArray:52 → folders: []
[DEBUG] 20:54:56.822 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 20:54:56.822 home.go:12 → /Users/valentino
[TRACE] 20:54:56.822 terminal.go:Flags() - 208ns
[DEBUG] 20:54:56.822 home.go:12 → /Users/valentino
[TRACE] 20:54:56.822 terminal.go:Flags() - 42ns
[TRACE] 20:54:56.822 terminal.go:GOOS() - 42ns
[DEBUG] 20:54:56.822 segment.go:Execute:112 → segment: Status
[DEBUG] 20:54:56.822 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 20:54:56.822 segment.go:isToggled:233 → no toggles found
[TRACE] 20:54:56.822 terminal.go:StatusCodes() - 83ns
[DEBUG] 20:54:56.822 properties.go:GetString:28 → status_template: {{ .Code }}
[TRACE] 20:54:56.822 text.go:Render(powerline) - 209ns
[TRACE] 20:54:56.822 text.go:Render(powerline) - 83ns
[DEBUG] 20:54:56.822 home.go:12 → /Users/valentino
[TRACE] 20:54:56.822 terminal.go:GOOS() - 42ns
[DEBUG] 20:54:56.822 segment.go:Execute:112 → segment: Julia
[DEBUG] 20:54:56.822 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 20:54:56.822 segment.go:isToggled:233 → no toggles found
[DEBUG] 20:54:56.822 properties.go:GetStringArray:52 → extensions: [*.jl]
[DEBUG] 20:54:56.822 properties.go:GetStringArray:52 → folders: []
[DEBUG] 20:54:56.822 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 20:54:56.822 home.go:12 → /Users/valentino
[TRACE] 20:54:56.822 text.go:Render(powerline) - 125ns
[DEBUG] 20:54:56.822 text.go:patchTemplate:125 → {{ .Code }}
[DEBUG] 20:54:56.823 properties.go:GetString:28 → display_mode: files
[DEBUG] 20:54:56.823 properties.go:GetStringArray:52 → extensions: [*.rb Rakefile Gemfile]
[DEBUG] 20:54:56.823 properties.go:GetStringArray:52 → folders: []
[DEBUG] 20:54:56.823 properties.go:GetStringArray:52 → extensions: [*.py *.ipynb pyproject.toml venv.bak]
[TRACE] 20:54:56.823 terminal_unix.go:Root() - 1.125µs
[DEBUG] 20:54:56.823 properties.go:GetStringArray:52 → folders: [.venv venv virtualenv venv-win pyenv-win]
[DEBUG] 20:54:56.823 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 20:54:56.823 properties.go:GetBool:22 → home_enabled: false
[DEBUG] 20:54:56.823 home.go:12 → /Users/valentino
[TRACE] 20:54:56.823 text.go:Render(powerline) - 125ns
[DEBUG] 20:54:56.823 home.go:12 → /Users/valentino
[TRACE] 20:54:56.823 text.go:Render(powerline) - 0s
[TRACE] 20:54:56.823 text.go:Render(powerline) - 42ns
[TRACE] 20:54:56.823 text.go:Render(powerline) - 41ns
[TRACE] 20:54:56.823 text.go:Render(powerline) - 42ns
[TRACE] 20:54:56.823 text.go:Render(powerline) - 42ns
[TRACE] 20:54:56.823 text.go:Render({{ .Code }}) - 554.667µs
[DEBUG] 20:54:56.823 properties.go:GetBool:22 → always_enabled: true
[DEBUG] 20:54:56.823 text.go:patchTemplate:125 →  {{ if gt .Code 0 }}{{ reason .Code }}{{ else }}{{ end }}
[TRACE] 20:54:56.823 text.go:Render( {{ if gt .Code 0 }}{{ reason .Code }}{{ else }}{{ end }} ) - 58.917µs
[DEBUG] 20:54:56.823 text.go:patchTemplate:125 → {{ if gt .Code 0 }}#cc2222{{ end }}
[TRACE] 20:54:56.823 text.go:Render({{ if gt .Code 0 }}#cc2222{{ end }}) - 14.458µs
[TRACE] 20:54:56.823 text.go:Render(powerline) - 83ns
[DEBUG] 20:54:56.823 text.go:patchTemplate:125 → {{ if gt .Code 0 }}#cc2222{{ end }}
[TRACE] 20:54:56.823 text.go:Render({{ if gt .Code 0 }}#cc2222{{ end }}) - 11.584µs
[DEBUG] 20:54:56.823 text.go:patchTemplate:125 →  {{ .Data.CurrentDate | date .Data.Format }} 
[TRACE] 20:54:56.823 text.go:Render( {{ .CurrentDate | date .Format }}  ) - 23.041µs
[TRACE] 20:54:56.823 text.go:Render(diamond) - 42ns
[DEBUG] 20:54:56.823 text.go:patchTemplate:125 → {{ if gt .Code 0 }}#cc2222{{ end }}
[TRACE] 20:54:56.823 text.go:Render({{ if gt .Code 0 }}#cc2222{{ end }}) - 9.875µs
[DEBUG] 20:54:56.826 terminal_unix.go:TerminalWidth:85 → terminal width: 239
[TRACE] 20:54:56.826 terminal_unix.go:TerminalWidth() - 3.096459ms
[TRACE] 20:54:56.826 terminal.go:Shell() - 125ns
[TRACE] 20:54:56.826 terminal.go:Shell() - 0s
[TRACE] 20:54:56.826 terminal.go:Flags() - 42ns
[DEBUG] 20:54:56.826 home.go:12 → /Users/valentino
[TRACE] 20:54:56.826 terminal.go:GOOS() - 41ns
[DEBUG] 20:54:56.826 segment.go:Execute:112 → segment: Text
[DEBUG] 20:54:56.826 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 20:54:56.826 segment.go:isToggled:233 → no toggles found
[TRACE] 20:54:56.826 text.go:Render(╰─) - 209ns
[TRACE] 20:54:56.826 text.go:Render(plain) - 42ns
[TRACE] 20:54:56.826 terminal.go:Shell() - 84ns
[TRACE] 20:54:56.826 terminal.go:Shell() - 42ns
[TRACE] 20:54:56.826 terminal.go:Flags() - 42ns
[DEBUG] 20:54:56.826 home.go:12 → /Users/valentino
[TRACE] 20:54:56.826 terminal.go:GOOS() - 41ns
[DEBUG] 20:54:56.826 segment.go:Execute:112 → segment: Text
[DEBUG] 20:54:56.826 file.go:Get:75 → cache key not found: toggle_cache
[DEBUG] 20:54:56.826 segment.go:isToggled:233 → no toggles found
[TRACE] 20:54:56.826 text.go:Render(─╯) - 167ns
[TRACE] 20:54:56.826 text.go:Render(plain) - 41ns
[TRACE] 20:54:56.826 terminal.go:Shell() - 42ns
[TRACE] 20:54:56.826 terminal.go:Shell() - 0s
[TRACE] 20:54:56.826 terminal.go:Flags() - 0s
[TRACE] 20:54:56.826 terminal.go:Shell() - 42ns
[DEBUG] 20:54:56.826 terminal_unix.go:TerminalWidth:65 → terminal width: 239
[TRACE] 20:54:56.826 terminal_unix.go:TerminalWidth() - 2.583µs
[DEBUG] 20:54:56.826 home.go:12 → /Users/valentino
[TRACE] 20:54:56.827 path.go:Path() - 17.584µs
[TRACE] 20:54:56.827 terminal.go:Flags() - 0s
```

--- a/website/docs/installation/customize.mdx
+++ b/website/docs/installation/customize.mdx
@@ -5,7 +5,7 @@ sidebar_label: 🧩 Customize
 ---
 
 import Tabs from "@theme/Tabs";
-import TabItem from "@theme/TabItem"
+import TabItem from "@theme/TabItem";
 
 The standard initialization sets Oh My Posh' default, built-in theme.
 
@@ -122,36 +122,15 @@ Once altered, reload your config for the changes to take effect.
 <TabItem value="nu">
 
 :::caution
-Oh My Posh requires Nushell v0.93.0 or higher.
+Oh My Posh requires Nushell `v0.104.0` or higher.
 :::
 
-Adjust the Oh My Posh init line in the Nushell env file (`$nu.env-path`) by adding the `--config` flag
+Adjust the Oh My Posh init line in the Nushell config file (`$nu.config-path`) by adding the `--config` flag
 with the location of your configuration.
 
 ```bash
 oh-my-posh init nu --config ~/jandedobbeleer.omp.json
 ```
-
-This saves the initialization script to `~/.oh-my-posh.nu`.
-Now, edit the Nushell config file (`$nu.config-path`) and add the following line at the bottom:
-
-```bash
-source ~/.oh-my-posh.nu
-```
-
-If you want to save the initialization script elsewhere, you can change the first line to something like this:
-
-```bash
-oh-my-posh init nu --config ~/jandedobbeleer.omp.json --print | save /mylocation/myscript.nu --force
-```
-
-And change the `source` line to:
-
-```bash
-source /mylocation/myscript.nu
-```
-
-Once altered, restart Nushell for the changes to take effect.
 
 </TabItem>
 <TabItem value="powershell">

--- a/website/docs/installation/prompt.mdx
+++ b/website/docs/installation/prompt.mdx
@@ -117,7 +117,7 @@ exec fish
 <TabItem value="nu">
 
 :::caution
-Oh My Posh requires Nushell v0.93.0 or higher.
+Oh My Posh requires Nushell `v0.104.0` or higher.
 :::
 
 Add the following snippet as the last line to the Nushell config file (`$nu.config-path`):

